test/karma.conf.js

```diff
@@ -66,14 +44,16 @@ module.exports = function(config) {
     // possible values: config.LOG_DISABLE || config.LOG_ERROR || config.LOG_WARN || config.LOG_INFO || config.LOG_DEBUG
     logLevel: config.LOG_INFO,
 
+    // test results reporter to use
+    reporters: ['progress'],
 
     // enable / disable watching file and executing tests whenever any file changes
-    autoWatch: true,
+    autoWatch: false,
```

Comment:
Will this still allow the tests to run when files are updated? 
Or do the tests have to be told to run?

Reply:
This forces you to execute testing after you make changes versus it auto doing it. I turned it off because in the master it should be off, but if your actively doing testing you can turn it on.

Reply:
Ahh I understand. That was my only question. This pull is approved from me.